root/components/forms.tt

```diff
@@ -306,6 +307,8 @@
           <option value="Turkish">[% l('Turkish') %]</option>
         </select>
         <br />
```

Comment:
This `<br />` seems like it should be removed now, so that the link is on the same line as in the case above.

Reply:
Removed from https://github.com/metabrainz/musicbrainz-server/pull/1179/commits/ee0f533c7ecfb8e664a23ef076835f1adbd050be as well.